src/java/picard/illumina/CollectIlluminaBasecallingMetrics.java

```diff
@@ -43,25 +43,49 @@
 import java.util.TreeMap;
 
 /***
- *  A Command line tool to collect Illumina Basecalling metrics for a sequencing run
- *  Requires a Lane and an input file of Barcodes to expect.
- *  Outputs metrics:
- *    *  Mean Clusters Per Tile
- *    *  Standard Deviation of Clusters Per Tile
- *    *  Mean Pf Clusters Per Tile
- *    *  Standard Deviation of Pf Clusters Per Tile
- *    *  Mean Percentage of Pf Clusters Per Tile
- *    *  Standard Deviation of Percentage of Pf Clusters Per Tile
- */
+        - *  A Command line tool to collect Illumina Basecalling metrics for a sequencing run
+        - *  Requires a Lane and an input file of Barcodes to expect.
+        - *  Outputs metrics:
+        - *    *  Mean Clusters Per Tile
+        - *    *  Standard Deviation of Clusters Per Tile
+        - *    *  Mean Pf Clusters Per Tile
+        - *    *  Standard Deviation of Pf Clusters Per Tile
+        - *    *  Mean Percentage of Pf Clusters Per Tile
+        - *    *  Standard Deviation of Percentage of Pf Clusters Per Tile
+        - */
+
 @CommandLineProgramProperties(
-        usage = CollectIlluminaBasecallingMetrics.USAGE,
-        usageShort = CollectIlluminaBasecallingMetrics.USAGE,
+        usage = CollectIlluminaBasecallingMetrics.USAGE_SUMMARY + CollectIlluminaBasecallingMetrics.USAGE_DETAILS,
+        usageShort = CollectIlluminaBasecallingMetrics.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class CollectIlluminaBasecallingMetrics extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Collects Illumina Basecalling metrics for a sequencing run.  ";
+    static final String USAGE_DETAILS = "<p>Program produces per-barcode and per-lane basecall metrics for each sequencing run.  Mean values for each metric are determined using data from all of the tiles.  This tool requires the LANE(#), BASECALLS_DIR, READ_STRUCTURE, and an input file listing the sample barcodes into the program.  " +
```

Comment:
I'd like to see all instances of sentences starting with "Program will [...]" replaced, because it's not proper English. I seem to have missed this one (and the following line) in my first pass. Something like "This tool will [...]" like we've used elsewhere should work. 

Also, make sure to wrap all lines so they're readily readable without scrolling in github.